specification/index.bs

```diff
@@ -149,12 +149,12 @@ to some portion of a <a>container document</a>.
 The present draft of this specification deals with two <a>score types</a>:
 
 <ul>
-  <li><a href="#cwmnx">CWMNX</a>, which encodes Conventional Western Music
+  <li><a href="#common">MNX-Common</a>, which encodes Conventional Western Music
   Notation (CWMN) using <a>semantic markup</a> as defined in the
   HTML5 standard, allowing applications to present the same information consistently in
   different contexts. It inherits many ideas and concepts from MusicXML.</li>
 
-  <li><a href="#gmnx">GMNX</a>, where "G" is for
+  <li><a href="#generic">MNX-Generic</a>, where "G" is for
```

Comment:
More specifically to @cecilios's comment, there is no more G for Generic or General in the name. So the first sentence can be removed and the "It" in the second sentence replaced with "MNX-Generic, which".